Makefile

```diff
@@ -163,7 +163,7 @@ ledger.docs: $(PDF_DIR)/cardano-ledger.pdf $(latexFiles)
 ledger.conway.docs: $(PDF_DIR)/conway-ledger.pdf
 docs: ledger.docs
 
-ledger.html: $(HTML_DIR)/$(LEDGER).PDF.html
+ledger.html: $(HTML_DIR)/Html.html
```

Comment:
Same comment: can we define `LEDGER = cardano-ledger` and make this line:
`ledger.html: $(HTML_DIR)/$(LEDGER).html`?